src/pytorch_lightning/loggers/csv_logs.py

```diff
@@ -169,7 +169,7 @@ def save_dir(self) -> Optional[str]:
         """
         return self._save_dir
 
-    @property
+    @property  # type: ignore
     @rank_zero_experiment
```

Comment:
is this because we depend on the other PR that updates the types for `rank_zero_experiment`?

Reply:
mypy throws an error that doesn't explicitly say anything about rank_zero_experiment; the error only states `Decorated property not supported`. I found this (somewhat dated) thread regarding the error, which gave the `# type: ignore as the solution` https://github.com/python/mypy/issues/1362